addrtoname.c

```diff
@@ -943,12 +943,12 @@ static const struct etherlist {
  * and etheraddr_string() fills in the table on demand.  If it doesn't,
  * then we suck in the entire /etc/ethers file at startup.  The idea
  * is that parsing the local file will be fast, but spinning through
- * all the ethers entries via NIS & next_etherent might be very slow.
+ * all the ethers entries via NIS & next_ethernet might be very slow.
```

Comment:
These changes are dependent on https://github.com/the-tcpdump-group/libpcap/pull/1198/commits/949aff552716fa5fe94dc10a1cb2bf9134bcacde in https://github.com/the-tcpdump-group/libpcap/pull/1198 and can be split out / dropped.

I'm going to drop them and address them separately.